lib/View.php

```diff
@@ -50,7 +50,7 @@ public function assign($name, $value)
      */
     public function draw($template)
```

Comment:
I doubt `draw` is the perfect term here, especially if the doc itself just says `render`; but I see this is off-topic.